packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -309,7 +310,7 @@ export class StorageService extends EventEmitter {
   }
 
   public async updateCommunityMetadata(communityMetadata: CommunityMetadata): Promise<CommunityMetadata | undefined> {
```

Comment:
[clarity/maybe bug?] It seems like currently the code in `CommunityMetadataStore` will discard the new metadata when there is an old entry with a `rootCa` and `ownerCertificate`. Should we fix this bug here, change the name of this function to something that suggests that this is a un-overwriteable, or put a comment in here that suggests that the functionality to update existing community metadata is not implemented yet? I'm fine with leaving that to another PR to get this through, but wanted to highlight that `updateCommunityMetadata` is misleading as is.
```ts
      // [Excerpt from CommunityMetadataStore]
      // FIXME: update community metadata if it has changed (so that
      // we can migrate community metadata easily)
      const oldMeta = this.getStore().get(newMeta.id)
      if (oldMeta?.ownerCertificate && oldMeta?.rootCa) {
        return oldMeta
      }
 ```